examples/client/client.c

```diff
@@ -365,108 +365,108 @@ static void SetKeyShare(WOLFSSL* ssl, int onlyKeyShare, int useX25519,
         if (useLibOqs) {
             int group = 0;
 
-            if (XSTRNCMP(oqsAlg, "KYBER512", XSTRLEN("KYBER512")) == 0) {
-                group = WOLFSSL_KYBER512;
+            if (XSTRNCMP(oqsAlg, "KYBER_LEVEL1", XSTRLEN("KYBER_LEVEL1")) == 0) {
```

Comment:
Is the term "LEVEL" being standardized to represent the strength or NIST rounds? Is there a way to cross reference these new LEVEL terms with the well understood "KYBER_512" for example?

Reply:
No, the term "level" is unlikely to be used. However, I wanted to use level just for that reason.  I don't want it to look like we're anticipating the naming convention in the upcoming standards.  

Yes.  Each of the papers of each of the algorithms lists their parameter set names and declares the claimed NIST level for that parameter set.  So, if you wanted to you would get the paper for that algorithm and check the level.

Reply:
By the way, here is a description of what each level means: 


    1. Any attack that breaks the relevant security definition must require computational resources comparable to or greater than those required for key search on a block cipher with a 128-bit key (e.g. AES128)
    2. Any attack that breaks the relevant security definition must require computational resources comparable to or greater than those required for collision search on a 256-bit hash function (e.g. SHA256/ SHA3-256)
    3. Any attack that breaks the relevant security definition must require computational resources comparable to or greater than those required for key search on a block cipher with a 192-bit key (e.g. AES192)
    4. Any attack that breaks the relevant security definition must require computational resources comparable to or greater than those required for collision search on a 384-bit hash function (e.g. SHA384/ SHA3-384)
    5. Any attack that breaks the relevant security definition must require computational resources comparable to or greater than those required for key search on a block cipher with a 256-bit key (e.g. AES 256)

...from https://csrc.nist.gov/projects/post-quantum-cryptography/post-quantum-cryptography-standardization/evaluation-criteria/security-(evaluation-criteria)

Reply:
Perhaps you can document these in brief somewhere in the header where these enums are defined?

Reply:
Yup.  Sure.  I can do that.